test/plugin/scenarios/vertx-web-3.x-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,93 @@
+registryItems:
+  applications:
```

Comment:
Hi @BFergerson, `expecteddata.yaml` is missed the licensed header, and you also missed the [github action configuration](https://github.com/apache/skywalking/blob/master/docs/en/guides/Plugin-test.md#local-test-and-pull-request-to-the-upstream)

Reply:
Thanks for taking a look. I added the missing license header but I didn't setup the GitHub action because I can't even get it to work locally yet. Locally I'm failing on the health check.

Reply:
Hi, we don't need this part anymore.